flow/packet-headers/icmpv6.yaml

```diff
@@ -7,311 +7,64 @@ components:
       properties:
         choice:
           type: string
-          enum: [dest_unreach, packet_too_big, time_exceeded, parameter_problem, echo_request, echo_reply, multicast_listner_query]
-        dest_unreach:
-          $ref: '#/components/schemas/Flow.ICMPv6DestUnreach'
-        packet_too_big:
-          $ref: '#/components/schemas/Flow.ICMPv6PacketTooBig'
-        time_exceeded:
-          $ref: '#/components/schemas/Flow.ICMPv6TimeExceeded'
-        parameter_problem:
-          $ref: '#/components/schemas/Flow.ICMPv6ParamProblem'
-        echo_request:
-          $ref: '#/components/schemas/Flow.ICMPv6EchoRequest'
-        echo_reply:
-          $ref: '#/components/schemas/Flow.ICMPv6EchoReply'
-        multicast_listner_query:
-          $ref: '#/components/schemas/Flow.ICMPv6MulticastListnerQuery'
+          enum: [echo]
+        echo:
+          $ref: '#/components/schemas/Flow.Icmpv6.Echo'
         
-    Flow.ICMPv6DestUnreach:
-      description: Packet Header for ICMPv6 Destination Unreachable Message
+    Flow.Icmpv6.Echo:
+      x-include: 
+      - '.#/components/schemas/Flow.Icmp.Common'
```

Comment:
this references the same name as in icmp.yaml - it should reference a unique name in this file
beginning . should be removed

Reply:
Changed the name